client/jetpack-cloud/components/user-feedback-form-modal/style.scss

```
@@ -0,0 +1,71 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.user-feedback-form-modal {
+	max-width: 827px;
+	max-height: 100%;
+	margin: 0;
+
+	.components-modal__content {
+		padding: 0;
+
+		> div {
+			display: flex;
+			flex-direction: column;
+			justify-content: space-between;
+			height: 100%;
+		}
+	}
+
+	.reviews-ratings-stars__star:focus {
+		outline: none;
+	}
+
+	@include break-medium {
+		margin: auto;
+	}
+}
+
+.user-feedback-form-modal .form-fieldset {
+	margin: 0;
+}
+
+.user-feedback-form-modal__main {
+	padding: 40px;
+	gap: 24px;
+	display: flex;
+	flex-direction: column;
+}
+
+.user-feedback-form-modal__title {
+	font-size: 1.5rem;
+	font-weight: 700;
+	line-height: 0.8;
+	margin: 0;
+	padding: 0;
+}
+
+p.user-feedback-form-modal__instruction {
+	font-size: 1rem;
+	font-weight: 400;
+	margin: 0;
+}
+
+.user-feedback-form-modal__footer {
+	padding: 24px 40px;
+	text-align: right;
+	background-color: var(--studio-gray-0);
+	border-radius: 4px;
+}
+
+.user-feedback-form-modal__close-button {
+	position: absolute;
+	right: 1rem;
+	top: 1rem;
```

Could we use RTL equivalent styles here?

```suggestion
	inset-inline-end: 1rem;
	inset-block-start: 1rem;
```